LEAF_Request_Portal/form.php

```diff
@@ -70,11 +69,10 @@ public function __construct($db, $login)
         {
             include __DIR__ . '/' . Config::$orgchartPath . '/sources/Group.php';
         }
-        $config = new Orgchart\Config;
-        $oc_db = new DB($config->dbHost, $config->dbUser, $config->dbPass, $config->dbName);
+        $oc_db = new DB($config->phonedbHost, $config->phonedbUser, $config->phonedbPass, $config->phonedbName);
         $oc_login = new OrgChart\Login($oc_db, $oc_db);
         $oc_login->loginUser();
-        $this->oc_dbName = $config->dbName;
+        $this->oc_dbName = $config->phonedbName;
```

Comment:
Is $config still defined, L73 ```$config = new Orgchart\Config;``` was removed.

Reply:
$config is still defined in LEAF/index.php, that's the reason that this was changed from dbName to phoneDBName.